includes/misc-functions.php

```diff
@@ -2624,117 +2624,3 @@ function give_get_page_by_title(string $page_title, string $output = OBJECT, str
 
     return get_post($pages[0], $output);
 }
-
- /**
- * @see https://github.com/jackocnr/intl-tel-input
- *
- * @unreleased
- */
-function give_get_intl_tel_input(string $value, string $id, string $class = '', string $name = ''):string {
```

Comment:
was this something that already existed or did you introduce it recently?  If it was here previously I would rather mark it as deprecated in case a third party is using.

Reply:
This was introduced in [this PR](https://github.com/impress-org/givewp/pull/7331) which was released in the Give 3.7 to be used in the donor profile page, so it's pretty recent. So, do you think still is necessary to deprecate it instead of just removing it?

Reply:
ah okay, in that case I think its safe to delete